docs/api-reference/api-completeness.md

```diff
@@ -0,0 +1,205 @@
+# API Completeness
```

Comment:
This file is generate by running:

```terminal
python utils/generate_backend_completeness.py
```

from the terminal.

I tried to make it a mkdocs plugin, but I failed without installing it. Maybe someone knows better, or we can add it to the pre-committer to run automatically.